### PR TITLE
Remove possible whitespace in title

### PR DESCRIPTION
**Pull request by pickfire**
image

It should fix the whitespace after regex and after platform, the first whitespace I thought I fixed it but it wasn't fixed.

### CODE REVIEW

**Comment by jyn514** [comment]
The current header looks like this:
image
I don't really see a difference between the two ...

**Comment by pickfire** [comment]
@jyn514 Right click on it, you will notice there is a bit of whitespace next to the bar, try selecting it (clicking or right clicking) since it will flip the color, then will see an obvious bar with some space next to it.

**Comment by jyn514** [comment]
It still looks the same to me:
Currently: 
image

Your changes: image

**Comment by pickfire** [comment]
This time should work except the first one still not fixed since tera have no way to remove the whitespace https://github.com/Keats/tera/issues/610, changing html is probably the only option right now.

image

File: templates/rustdoc/topbar.html [hunk 1]
```diff
@@ -162,16 +162,16 @@
...
             <a href="{{ crate_url | safe }}" class="pure-menu-link crate-name" title="{{ metadata.description }}">
                 {{ "cube" | fas }}
                 <span class="title">{{ metadata.name }}-{{ metadata.version }}</span>
             </a>
         </li>
-    {%- endif -%}
-
-    {# If this is the latest release and it's been yanked, just display a warning #}
+    {%- endif -%}{#
+
```
Comment (jyn514) [inline templates/rustdoc/topbar.html:173]: Can you remove this extra whitespace at the end of the line?

File: templates/rustdoc/topbar.html [hunk 2]
```diff
@@ -202,9 +202,9 @@
             </a>
         </li>
     {%- endif -%}
-
+
```
Comment (jyn514) [inline templates/rustdoc/topbar.html:205]: Same here.

**Review by jyn514 — COMMENTED** [review]
LGTM with the whitespace fixed.

File: templates/rustdoc/topbar.html (hunk 2 above)
Comment (pickfire) [reply templates/rustdoc/topbar.html]: Done, thanks.

**Pull request merged**